presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -361,7 +361,7 @@ public void testExplainAnalyzeDDL()
         computeActual("EXPLAIN ANALYZE DROP TABLE orders");
     }
 
-    private void assertExplainAnalyze(@Language("SQL") String query)
+    protected void assertExplainAnalyze(@Language("SQL") String query)
```

Comment:
Why would you need to change it from `private` to `protected`? Could you run the tests without this change?

Reply:
I updated this so I could call this method in the test. e.g. here: https://github.com/prestodb/presto/pull/20776/files#diff-e86735ad8b4b0dd7856263d9696e731dd986026f5547b8f78265e1e389a308daR99

Reply:
I assume you changed the visibility because this PR only partially supports the base `testDelete` function. I don't suggest updating the visibility due to partial support of the deletion feature.
Would you happen to have a plan to support other deletion syntaxes?

Reply:
Because the function was designed to be `private`. If this PR supports the full `testDelete` function, we probably wouldn't need to update the visibility as we could just call the base `testDelete` function, right?

Reply:
That's also why I asked about your plan to support other deletion syntaxes.

Reply:
The plan for supporting the rest of the delete syntax is going to be up to the Presto's iceberg contributors. The next step is probably to support whole-partition deletes, but I don't have the intention of including that in this PR. 

I've changed the visibility back on this method